### ISSUE TITLE
Only One Device Found When Multiple Are Defined

### ISSUE THREAD

**Issue by StuBFrost** [p10tyr/rtsp-to-onvif#15]
Hi.  I have a Swann DVR that has 5 cameras all on the same IP address.  

When I set up the config.yaml file I name the cameras "Camera1", "Camera2", "Camera3" etc.  

Protect only finds one camera and it is always "Camera5" but strangely the model is "Camera4".

Could my yaml file be an issue?  I don't have a lot of experience with these things.

Thanks

**Comment by balgerion** [comment]
Did you figure out what's going on ? i have similar situation. For me two or three cameras in one yaml config are working and others not .

**Comment by StuBFrost** [comment]
> Did you figure out what's going on ? i have similar situation. For me two or three cameras in one yaml config are working and others not .

No. I had to stop using the server as I kept getting the issue "EADDRINUSE crash after several hours" which is issue 10 and this project appears to be abandoned.

**Comment by tmushy** [comment]
Anyone ever get this working. With one camera I can get it working fine but as soon as I start adding more I get issues described in the post
